### PR TITLE
Broken Link Fixes 

### PR DESCRIPTION
**Pull request by ssavi-ict**
e.g. Fixes #0 

**Checklist**:

- [x] The GitHub pipeline is OK (green), 
      meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.

- [x] A unit test is covering the code added / modified by this PR

- [x] This PR is ready to be merged 

- [x] In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder

- [ ] A mention of the change is present in `CHANGELOG.md`

By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/PyFPDF/fpdf2/blob/master/LICENSE).

### CODE REVIEW

**Review by Lucas-C — APPROVED** [review]
Thank you @ssavi-ict!

**Comment by Lucas-C** [comment]
Could you also include in this PR the fixes for the broken tutorial links in `docs/index.md`, please? 😊
https://pyfpdf.github.io/fpdf2/index.html#tutorials

**Comment by ssavi-ict** [comment]
@Lucas-C , the first broken link I noticed was this one. Later I forgot where I have seen that. Thanks for noticing this issue. Edited accordingly.

File: docs/index.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Go try it **now** online in a Jupyter notebook: [![Open In Colab](https://colab.
 * [Ελληνικά](Tutorial-gr.md)
 * [עברית](Tutorial-he.md)
 * [简体中文](Tutorial-zh.md)
-* [বাংলা](Tutorial-%E0%A6%AC%E0%A6%BE%E0%A6%82%E0%A6%B2%E0%A6%BE.md)
```
Comment (Lucas-C) [inline docs/index.md:65]: Thanks!

There is also another broken link on line 59, a little above this one 😅
`Tutorial-हिंदी.md` should be renamed. Could you please fix it too?

Comment (ssavi-ict) [reply docs/index.md:65]: :P .. Missed it again. Thank You. Changes pushed.

**Comment by ssavi-ict** [comment]
@Lucas-C , Will it be good if we mention when a translator would like to translate in his/her own language (s)he needs to -
- Reflect the changes into `index.md`, `CHANGELOG.md`, `README.md`, `mkdocs.yml` files.
- The translation file name should be written in English letter. 

Maybe you can edit issue #267 with some guidelines like this. 
This will be helpful for an OPEN SOURCE beginner like me.

**Comment by Lucas-C** [comment]
> @Lucas-C , Will it be good if we mention when a translator would like to translate in his/her own language (s)he needs to -
> 
>     * Reflect the changes into `index.md`, `CHANGELOG.md`, `README.md`, `mkdocs.yml` files with line numbers or section notation.
> 
>     * The translation file name should be written in English letter.
> 
> 
> Maybe you can edit issue #267 with some guidelines like this. This will be helpful for an OPEN SOURCE beginner like me.

Excellent idea!
I have just done so 😊

**Comment by ssavi-ict** [comment]
> Excellent idea! I have just done so 😊

Great. Thank You ...

**Pull request merged**